src/map/storage.c

```diff
@@ -887,6 +993,145 @@ static int storage_guild_storage_quit(struct map_session_data *sd, int flag)
 	return 0;
 }
 
+/**
+ * Read additional storage configuration fields for plugins.
+ * 
+ * @param t          [in]    pointer to the config element being parsed.
+ * @param s_conf     [in]    pointer to the config struct being written to.
+ * @param filename   [in]    pointer to the filename string.
+ */
+static void storage_config_read_additional_fields(struct config_setting_t* t, struct storage_settings* s_conf, const char* filename)
+{
+	// plugins do their own thing.
+}
+
+/**
+ * Dynamically reads storage configuration and initializes required variables.
+ *
+ * @param filename Path to configuration file.
+ * @param imported Whether the current config is imported from another file.
+ * @retval false in case of error.
+ */
+bool storage_config_read(const char* filename, bool imported)
+{
+	nullpo_retr(false, filename);
+
+	if (!imported)
+		VECTOR_INIT(storage->configuration);
```

Comment:
I've been thinking. I wanna drop this feature altogether. Since `storage.conf` will be as is almost forever officially similar to other configs. There is no point having additional fields and imports.

Reply:
9f08c77a4a4a41c9ea0bd85a9ff678043961edb0

I transferred config read function below the initialization.